src/pages/Tags/Tags.tsx

```diff
@@ -1,9 +1,242 @@
+      },
+    ])
+  }
+
+  const onTagsSubmit: SubmitHandler<FormValues> = useCallback(
+    async (values) => {
+      const { tagCategorySelection, tagInput } = values
+      const transformedObject = flatMap(tagInput, (tagInputValue) =>
+        map(tagCategorySelection, (tagCategoryValue) => {
+          return {
+            type: tagCategoryValue,
+            name: tagInputValue,
+          }
+        })
+      )
+      const payload: TagsType = {
+        tags: transformedObject,
+      }
+
+      try {
+        await createTags(payload)
+        showNotification({
+          type: NotificationTypes.Success,
+          title: t('notification.announcement'),
+          content: t('success.tag_added'),
+        })
+        reset()
+      } catch (errorData) {
+        showValidationErrorMessage(errorData)
+      }
+    },
+    [createTags, reset, t]
+  )
+
+  const groupedCategoryData = groupBy(tags, 'type')
+  const uniqueCategoryTypes = Object.keys(groupedCategoryData)
+
+  const areInputFieldsDirty = !!(
+    dirtyFields.tagCategorySelection && dirtyFields.tagInput
+  )
+
+  if (isFetchingTags) {
+    return <Loader loading />
+  }
+
+  return (
+    <>
+      <div className={classes.tagsHeader}>
+        <h1>{t('tag.tag_management')}</h1>
+        <Tooltip
+          title={t('cheat_sheet.user_management.title')}
+          modalContent={<TagsCheatSheet />}
+        />
+      </div>
+      <Container className={classes.tagsContainer}>
+        <div>
+          <h4 className={classes.addingTag}>{t('tag.adding_tag')}</h4>
+          <p>{t('tag.naming_tag')}</p>
+        </div>
+
+        <div className={classes.addingTagsSeparator} />
+
+        <div className={classes.tagsSection}>
+          <DynamicForm fields={inputFields} control={control} />
+          <Button
+            appearance={AppearanceTypes.Text}
+            iconPositioning={IconPositioningTypes.Left}
+            icon={Add}
+            className={classes.addNewRow}
+            children={t('tag.add_new_row')}
+            onClick={addInputField}
+            hidden={!includes(userPrivileges, Privileges.AddTag)}
+          />
+        </div>
+
+        <div className={classes.categorySection}>
+          <DynamicForm fields={categoryFields} control={control} />
+          <Button
+            children={t('button.add')}
+            type="submit"
+            ariaLabel={t('button.add')}
+            className={classes.addButton}
+            onClick={handleSubmit(onTagsSubmit)}
+            loading={isCreatingTags}
+            disabled={!areInputFieldsDirty}
+            hidden={!includes(userPrivileges, Privileges.AddTag)}
+          />
+        </div>
+      </Container>
+
+      <div className={classes.categoryContainer}>
+        {map(uniqueCategoryTypes, (type) => (
+          <Container key={type} className={classes.category}>
+            <span className={classes.tagCategoryNameContainer}>
+              <div className={classes.categoryName}>{type}</div>
```

Comment:
**[blocker]**

1. Text should always be in the appropriate html tags. div is not meant for text. 
2. You are currently displaying the "type" value as text to the user, however this needs to be translatable. (Everything displayed to the user should be translatable).
3. You are using "span" tag as a container for a div + a button, however <span> tag is meant for text